.github/workflows/whitesource.yml

```diff
@@ -24,6 +24,6 @@ jobs:
     - name: PR
       uses: ./whitesource-actions/whitesource-pr
       with:
-        jira_token: ${{ secrets.JIRA_TOKEN }}
+        jira_token: ${{ secrets.JIRA_TOKEN_PUBLIC_REPO }}
```

Comment:
I am assuming this is already added somewhere?

Reply:
never mind i see those in secrets of this repo inherited from parent (snowflakedb)